plugins/analytics-module-ga/config.d.ts

```diff
@@ -34,6 +34,15 @@ export interface Config {
          */
         scriptSrc?: string;
 
+        /**
+         * When set to `true`, all pageviews and hits are deferred until an
+         * identity is known. Guarantees that all data sent to GA correlates
+         * to a user identity; prevents GA from ever receiving events for
+         * sessions in which a user does not sign in.
+         * @visibility frontend
+         */
+        requireIdentity?: boolean;
```

Comment:
```suggestion
        identity?: 'required' | 'optional' | 'disabled';
```

or something like that? more evolvable